src/libraries/oob-src.proj

```diff
@@ -21,8 +21,7 @@
                                                                                                             ('$(BuildAllConfigurations)' != 'true' and '$(RuntimeFlavor)' == '$(PrimaryRuntimeFlavor)')" />
 
     <!-- Don't build task and tools project in the NetCoreAppCurrent vertical. -->
-    <ProjectReference Remove="Microsoft.NETCore.Platforms\src\Microsoft.NETCore.Platforms.csproj;
-                              Microsoft.XmlSerializer.Generator\src\Microsoft.XmlSerializer.Generator.csproj" />
+    <ProjectReference Remove="Microsoft.XmlSerializer.Generator\src\Microsoft.XmlSerializer.Generator.csproj" />
```

Comment:
@ViktorHofer should I make a similar change to `oob-all.proj`?

Reply:
This change will cause the Microsoft.NETCore.Platforms.csproj package to be created and uploaded by every single vertical which will break official builds (as the publish task won't know which `Microsoft.NETCore.Platforms.csproj` package to pick).

If you want to preserve the existing behavior, make sure that only the allconfigurations leg produces the package. You can do that by i.e. specifying the following in the Platforms.csproj:

```
<!-- Generate a package only during the allconfigurations leg to avoid package clashes during publish. -->
<GeneratePackageOnBuild Condition="'$(BuildAllConfigurations)' != 'true'">false</GeneratePackageOnBuild>
```

Reply:
> @ViktorHofer should I make a similar change to oob-all.proj?

And yes, you want to remove https://github.com/dotnet/runtime/blob/19c5d17a5d71134bd774496a80d43fe13184ba09/src/libraries/oob-all.proj#L20